frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1162,14 +1220,14 @@ impl<T: Config> Pallet<T> {
 	/// 1. Increment round.
 	/// 2. Change phase to [`Phase::Off`]
 	/// 3. Clear all snapshot data.
-	fn post_elect() {
-		// inc round
+	fn rotate_round() {
+		// inc round.
```

Comment:
Comments should start with a capital letter to match rest of the source (here and after).

Reply:
yes I fixed here and in staking